mmv1/products/datastream/Stream.yaml

```diff
@@ -1149,6 +1374,94 @@ properties:
                               output: true
                               description: |
                                 The ordinal position of the column in the table.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'sqlServerExcludedObjects'
+        min_version: beta
```

Comment:
Seems like this is not beta-only

```suggestion
```

Reply:
Removed in 939b4f1e8962c3ac939f24d2fc98170f48bdc4d7